src/main/java/org/opensearch/sdk/NettyTransport.java

```diff
@@ -40,9 +40,23 @@
 
 public class NettyTransport {
     private static final String NODE_NAME_SETTING = "node.name";
+    private ExtensionsRunner extensionsRunner;
```

Comment:
Suggest making this final. We want to guarantee that it's initialized in the constructor, and there will only ever be one runner set.